.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,10 +1,7 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-
----
-
-## Bug Description
+---## Bug Description
```

Comment:
@obetomuniz Can you undo the changes to the Markdown files within the `.github` folder?

Not 100% sure how that happened, but if it's an issue with the pre-commit hook I'd need to fix that independently.

Reply:
^ Or @ndev1991

Reply:
@swissspidy It seems happened while merging master branch to here. Let me fix this

Reply:
@swissspidy Fixed. FYI, empty line added before ## with the pre-commit.

Reply:
Thanks. I meant all files in `.github/` , not just the one file. Did that now in f17477041c5404476ba5a838261f97a71570aa3c.

Will investigate further why this happened.